x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agents/agent_list_page/components/status_badges.tsx

```diff
@@ -26,11 +26,14 @@ export const AgentStatusBadges: React.FC<{
 }> = memo(({ agentStatus }) => {
   return (
     <EuiFlexGroup gutterSize="m">
-      {AGENT_STATUSES.map((status) => (
-        <EuiFlexItem key={status} grow={false}>
-          <AgentStatusBadge status={status} count={agentStatus[status] || 0} />
-        </EuiFlexItem>
-      ))}
+      {AGENT_STATUSES.map(
+        (status) =>
+          agentStatus[status] > 0 && (
```

Comment:
I am not sure we want to filter if there is no agent status, we recently changed that https://github.com/elastic/kibana/pull/191535

Reply:
Removed in https://github.com/elastic/kibana/pull/205815/commits/3fa3049e018dcec29f07415a78ea581300c2a1b2